src/test/java/com/scottlogic/ContentPostSorterTest.java

```diff
@@ -0,0 +1,119 @@
+package com.scottlogic;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.List;
+
+public class ContentPostSorterTest {
+
+    ContentPostSorter sortByContent = new ContentPostSorter();
+
+    private UserPost createUserPost(String content) {
+
+        return new UserPost("Joe Bloggs",
+                OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+                content, 2);
+    }
+
+    @Test
+    public void sort_mixedLengthPostsAsc_correctListReturned() {
+        UserPost userPost1 = createUserPost("An example of a post \nwith lines breaks.");
+        UserPost userPost2 = createUserPost("Another example post.");
+        UserPost userPost3 = createUserPost("Hello World!");
+
+        List<UserPost> userPosts = Arrays.asList(userPost1, userPost2, userPost3);
+        List<UserPost> sortByMethod = sortByContent.sort(userPosts, SortOrder.ASC);
+        List<UserPost> sorted = Arrays.asList(userPost3, userPost2, userPost1);
+        Assert.assertEquals(sortByMethod, sorted);
+    }
+
+    @Test
+    public void sort_sameLengthPostsAsc_correctListReturned() {
```

Comment:
What is the expected outcome when there are two posts of the same length? Is the tie broken on another criteria, or does it maintain the input order? Is it any different when sorting in descending order?

Reply:
I'm not sure if it hasn't updated on github. I've added testcases for this scenario. If the length is the same, it maintains input order for both ascending and descending.